src/utils/settings.cpp

```diff
@@ -42,11 +44,17 @@ const auto client_root = QStringLiteral("client");
 const auto petenv_name = QStringLiteral("primary");
 const auto autostart_default = QStringLiteral("true");
 
+QString default_hotkey()
+{
+    return QKeySequence{mp::hotkey_default}.toString(QKeySequence::NativeText); // outcome depends on platform
```

Comment:
Oh so will this translate to "Cmd" and "Opt" on macOS?

Reply:
Yup :wink: Well, actually to ⌘⌥U